types/gestalt/index.d.ts

```diff
@@ -718,11 +756,29 @@ export interface ModalProps {
     closeOnOutsideClick?: boolean;
     footer?: React.ReactNode;
     heading?: string | React.ReactNode;
-    ref?: React.Ref<'div'>;
     role?: 'alertdialog' | 'dialog';
     size?: 'sm' | 'md' | 'lg' | number;
 }
 
+/*
```

Comment:
just FYI, these comments won't be shown in intellisense unless they're jsdoc comments, i.e. they start with `/**` not `/*`

Reply:
thanks, added jsdoc formatting for all props